Flu-Vacc/README.md

```diff
@@ -0,0 +1,62 @@
+# SynthPop FluVacc Model
+
+## Authors
+
+Noah Burrell (noah.burrell@epistemix.com)
+
+## Description
+
+This model is an adaptation of a model originally implemented using NetLogo. The NetLogo implementation provides the following description:
```

Comment:
Was the quoted text below from a website or a paper? If a website, I suggest linking to that site from the text `NetLogo implementation`. If a paper, a citation would be appropriate here. This is out of an abundance of caution relating to the additional criteria noted in the PR description.

Reply:
The quoted text is from the NetLogo model itself (the actual `.nlogo` file), which has a description included. AFAIK, that file does not exist somewhere that I could link to it, it was shared with me via email. But I can add citations to the paper in the first sentence and clarify the second sentence with something like "alongside the code"

Reply:
Perfect